cypress/integration/search_for_records.ts

```diff
@@ -0,0 +1,25 @@
+import { AuthRoles } from '../support/commands';
+
+describe('Search for records by person', () => {
+  describe('As a user in the Admin Dev group', () => {
+    it('Search of Adult group record', () => {
+      cy.visitAs('/cases', AuthRoles.AdminDevGroup);
+
+      cy.contains('First name:').type(Cypress.env('ADULT_RECORD_FIRST_NAME'));
```

Comment:
How would Cypress know this would be an Adult first name? 
Is there some hidden test setup? 🤔 
I am assuming `contains` is an assertion?

Reply:
So this is saying:
1. Get the element that contains `First name:` (this is likely a text input)
2. Type the value of the environment variable `CYPRESS_ ADULT_RECORD_FIRST_NAME` into that element

Reply:
Ah... I thought `.type` checked the `TYPE` of an object and not type a value into the element 😅

Reply:
Ah yeah, Cypress fun! Docs here: https://docs.cypress.io/api/commands/type

It's a bit wild at times, I find myself checking the docs quite often hah!

Reply:
🙌🏾